config/software/openssl-windows.rb

```diff
@@ -65,35 +65,18 @@
   end
 end
 
-build do
-  env = with_standard_compiler_flags(with_embedded_path)
+relative_path 'bin'
```

Comment:
Although this is a valid workaround for the current issue it still worries me. Mainly because multiple software components are being extracted into the same directory. We should get a fix into core Omnibus that guarantees each software component is extracted into a clean directory by default.

For background the core issue is knap-build produced archives don't follow the name-version/ toplevel directory convention of regular tarballs. `Omnibus::NetFetcher` assumes this convention in its extraction logic. This means the libyaml-windows files are extracted to the root of `C:\omnibus-ruby\src` vs the expected `C:\omnibus-ruby\src\<SOFTWARE>-<VERSION>`.